src/lib/override/adopted-probes.ts

```diff
@@ -338,12 +349,28 @@ export class AdoptedProbes {
 		}
 	}
 
-	private async updateIds (currentAdoptedIp: string, uuid: string | null, connectedProbe: Probe) {
-		await this.sql(ADOPTED_PROBES_TABLE).where({ ip: currentAdoptedIp }).update({
-			ip: connectedProbe.ipAddress,
-			altIps: JSON.stringify(connectedProbe.altIpAddresses),
-			...connectedProbe.uuid !== uuid ? { uuid: connectedProbe.uuid } : {},
-		});
+	private async updateIds (currentAdoptedIp: string, connectedProbe: Probe) {
+		const update = async () => {
+			await this.sql(ADOPTED_PROBES_TABLE).where({ ip: currentAdoptedIp }).update({
+				ip: connectedProbe.ipAddress,
+				altIps: JSON.stringify(connectedProbe.altIpAddresses),
+				uuid: connectedProbe.uuid,
```

Comment:
I don't think including current uuid of the probe into the update body was ever a problem. I tried that manually on staging and such updates were working fine. The real reason for `Duplicate entry 'zzz' for key 'gp_adopted_probes_uuid_unique'` is the same as for duplication of ip, just vice versa: old adoption tries to sync uuid by the ip, but that uuid is already occupied by the new adoption.

Reply:
I can't replicate it manually either, but consider that #570 only changed the behavior when the UUID stayed the same and it did fix the issue. So:
 - If the UUID changed the update happens as before.
 - If the UUID didn't change, that means the probe already has that UUID in the DB. The UUID can't be in any other row since it's unique.

Maybe the catch could be in the part *If the UUID didn't change, that means the probe already has that UUID in the DB* because we're processing updates for 8 probes in parallel, and the probe objects are shared on the node side, so it's possible we did some changes in the memory which are not applied in the DB yet, but I can't think of the exact case that would lead to this error.

Anyway, let's keep the PR as it is I guess, but extend this log at least to log alt IPs and UUIDs too - in case we find there's still some problem, it'll be easier to understand:

```
logger.warn('Deleting probes with ids:', excessProbesIds);
```